tritium-lib/src/test/java/com/palantir/tritium/proxy/InvocationEventProxyTest.java

```diff
@@ -279,7 +279,7 @@ public void testThrowingHandler() throws Throwable {
         assertThat(proxy.invoke(mockHandler, getStringLengthMethod(), EMPTY_ARGS)).isEqualTo("test".length());
 
         verify(mockHandler).isEnabled();
-        verifyZeroInteractions(mockFilter);
+        verifyNoInteractions(mockFilter);
```

Comment:
`verifyZeroInteractions` is deprecated now

Reply:
Sounds like a good candidate for some refastering ;)

Reply:
Yeah, this was the only deprecated occurrence in tritium so one off edit was faster, but providing `refaster`s in baseline for common libs like Mockito, Guava, etc. seems beneficial for the fleet (a quick search of `verifyZeroInteractions` shows many hundreds of occurrences).

I don't think I'll have cycles to write up the refaster rule in short term, so if you all want to own it that'd be great 😄

Reply:
Gave it a shot to play with refaster: https://github.com/palantir/gradle-baseline/pull/924